client/my-sites/sharing/style.scss

```diff
@@ -548,7 +548,7 @@
 	margin-bottom: 20px;
 	padding: 20px 24px;
 	background: var( --color-white );
-	box-shadow: 0 0 0 1px rgba( var( --color-neutral-100-rgb ), 0.5 ), 0 1px 2px var( --color-neutral-0 );
+	box-shadow: 0 0 0 1px rgba( lighten( $gray, 20% ), 0.5 ), 0 1px 2px var( --color-neutral-0 );
```

Comment:
I don't think we should be using SASS colour variables now that we have themes. cc @flootr

Reply:
You're right. Good eye. Thanks for calling this out. Reverted in https://github.com/Automattic/wp-calypso/commit/defda7181fc6d34534d46798b5211444e8c180f2